e2e/atlas_alerts_test.go

```diff
@@ -104,8 +104,8 @@ func TestAtlasAlerts(t *testing.T) {
 			t.Fatalf("unexpected error: %v", err)
 		}
 
-		if len(alerts) != 7 {
-			t.Errorf("got=%#v\nwant=%#v\n", len(alerts), 7)
+		if len(alerts) != 14 {
```

Comment:
we need to find a solution for this as this may keep growing, maybe for now change to not be zero(0)?